addons/binding/org.openhab.binding.smhi/src/main/java/org/openhab/binding/smhi/internal/model/SmhiData.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.smhi.internal.model;
+
+import java.util.List;
+
+/**
+ * The {@link SmhiData} is the Java class used to map the JSON response to an SMHI
+ * request.
+ *
+ * @author Michael Parment - Initial contribution
+ */
+
+public class SmhiData {
+    public List<SmhiTimeSeries> timeSeries;
```

Comment:
is it necessary that this is `public`? wouldn't a setter be more consistent?

Reply:
Fixed